### PR TITLE
Removing the ossec word from all the rules and unifying ossec and wazuh rules' files

### PR DESCRIPTION
**Pull request by DProvinciani**
This pull request implements all the necessary changes to remove the `ossec` word from the rules. For more details, check the description of the issue #788.

Is important to mention that some rules are still looking for the word `ossec` in its regex, as well as `wazuh`. This is because we are trying to keep retro compatibility between new managers and old agents registered to them.

### CODE REVIEW

File: rules/0015-wazuh_rules.xml [hunk 1]
```diff
@@ -1,63 +1,229 @@
 <!--
-  -  OSSEC rules
-  -  Author: Daniel Cid.
-  -  Updated by Wazuh, Inc.
+  -  Wazuh rules
+  -  Created by Wazuh, Inc.
```
Comment (pereyra-m) [inline rules/0015-wazuh_rules.xml:3]: We should still mention _OSSEC_ in the file header.

File: rules/0015-wazuh_rules.xml [hunk 2]
```diff
@@ -315,7 +481,7 @@
   </rule>
 
   <rule id="598" level="5">
-    <category>ossec</category>
+    <category>wazuh</category>
     <if_sid>554</if_sid>
     <hostname>syscheck-registry</hostname>
     <group>syscheck,pci_dss_11.5,gpg13_4.13,gdpr_II_5.1.f,hipaa_164.312.c.1,hipaa_164.312.c.2,nist_800_53_SI.7,tsc_PI1.4,tsc_PI1.5,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
```
Comment (pereyra-m) [inline rules/0015-wazuh_rules.xml:487]: We could maybe replace the installation path in the example (line 493): `/var/ossec/active-response/bin/firewall-drop.sh delete` for something like: /INSTALLATION_PATH/active-response/bin/firewall-drop.sh delete

**Review by pereyra-m — COMMENTED** [review]
LGTM!
We still need to rebase the other PR and wait for Jenkins

File: rules/0015-wazuh_rules.xml (hunk 1 above)
Comment (palaciosjeremias) [reply rules/0015-wazuh_rules.xml:3]: Maybe now that file name has changed, we should name this like i.e. `Wazuh rules Created by Wazuh, Inc. Based on OSSEC rules, credits to Daniel Cid.`

**Review by palaciosjeremias — APPROVED** [review]
LGTM

File: rules/0015-wazuh_rules.xml (hunk 1 above)
Comment (DProvinciani) [reply rules/0015-wazuh_rules.xml:3]: Fixed.

File: rules/0015-wazuh_rules.xml (hunk 2 above)
Comment (DProvinciani) [reply rules/0015-wazuh_rules.xml]: Fixed

**Pull request merged**